examples/docker/census-on-omnisci/census-omnisci.dockerfile

```diff
@@ -49,16 +46,12 @@ RUN conda update -n base -c defaults conda -y && \
     conda config --set channel_priority strict && \
     conda create -n modin --yes --no-default-packages && \
     conda activate modin && \
-    conda install -c intel/label/modin -c conda-forge modin "ray>=1.0.0" "numpy<1.20.0"
-
-RUN conda activate modin && \
-    conda install -c intel/label/modin -c conda-forge -c intel ${add_extra_channel} \
-        "daal4py>=2021.1" dpcpp_cpp_rt && \
-    conda install -c conda-forge "scikit-learn<0.24.0" && \
+    conda install -c intel/label/modin -c conda-forge -c intel \
+        modin "ray>=1.0.0" "numpy<1.20.0" "scikit-learn<0.24.0" "daal4py>=2021.1" dpcpp_cpp_rt && \
```

Comment:
Creating the environment with two separate commands (`conda install`) was significantly faster than installing with one command. 
How long does it take to create the environment now?

Reply:
I tested that 2nd call to conda install is much slower, and 3rd call to conda install finds some conflicts, tries to resolve them and never ends at all. So it is much faster to do in one call. It takes less than 10 minutes to complete after #3052 was merged.

Reply:
Good!